libs/sdk-ui-kit/styles/scss/header.scss

```diff
@@ -584,8 +585,34 @@ $gd-header-project-width: 230px;
     }
 }
 
+.project-picker-header {
+    margin: 17px 15px 0px 15px;
+    font-size: 11px;
+    font-weight: bold;
+    text-transform: uppercase;
+    color: #94a1ad;
+    position: relative;
+    line-height: 11px;
+    overflow: hidden;
+
+    &::after {
+        content: "";
+        position: absolute;
+        margin-top: 5px;
+        display: inline-block;
+        width: 100%;
+        height: 0;
+        margin-left: 10px;
+        border-top: 1px solid #dde4eb;
+    }
+}
+
 .gd-header-project-picker-footer {
-    margin: 0 11px;
-    padding: 11px 0;
+    padding: 12px 15px;
     border-top: 1px solid $gd-border-color;
-}
+
+    .home-footer-icon {
+        background: url("assets/home.svg") no-repeat;
+        padding-left: 30px;
+    }
+}
```

Comment:
You can enable automatic newline in your editor. All the well known supports it ;)